Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -353,7 +353,16 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
       }
 
     let checkoutValues = createCheckoutEvents.values()
-      .map { $0.checkout.id }
+      .map { values in
+        var checkoutId: String?
+
+        if let decoded = decodeBase64(values.checkout.id), let range = decoded.range(of: "Checkout-") {
+          let id = decoded[range.upperBound...]
+          checkoutId = String(id)
+        }
+
+        return checkoutId
+      }
```

Comment:
These will always decode to include "Checkout-", but I'm not a big fan of hardcoding strings like this

Reply:
Huh, is this decoding a GraphID sent back from GraphQL? You should ask the server-side folks to give you a plain ID in the mutation. Those Base64 IDs are sort of intended to be opaque and shouldn't need to be read like this.